server/sonar-web/src/main/js/api/settings.js

```diff
@@ -93,3 +93,8 @@ export function getServerId () {
 export function generateServerId (organization, ip) {
   return postJSON('/api/server_id/generate', { organization, ip });
 }
+
+// TODO replace with /api/settings
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1135)